src/Components/Charts/Bar.tsx

```diff
@@ -0,0 +1,387 @@
+import React, { useContext, useEffect, useState } from "react";
+import { ResponsiveBar } from '@nivo/bar'
+import axios from "axios";
+import { AuthContext } from "../../Context/AuthContext";
+
+import { ResponsiveStream } from '@nivo/stream'
+export default function Bar() {
+  const { baseUrl, requstHeaders }: any = useContext(AuthContext);
+  const [contTasks, setContTasks] = useState([]);
+  const [contUsers, setcontUsers] = useState([]);
+  const getContTasks = () => {
+    axios
+      .get(`${baseUrl}/Task/count`, {
+        headers: requstHeaders,
+      })
+      .then((response) => {
+        console.log(response);
```

Comment:
consider removing logs before making a pull request, to avoid merging to the production branch (master) with logs in the user's browser, and these logs might reveal critical information.